galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos.manager/src/main/java/dev/galasa/zosbatch/IZosBatchJob.java

```diff
@@ -92,7 +92,7 @@ public static JobStatus valueOfLabel(String jobStatus) {
     
     /**
      * The batch job completion return code, e.g.<br>
-     * <code>CC 0000</code>, <code>CC 0020</code>, <code>JCL ERROR</code>, <code>ABEND S0C4/code> etc.<br>
+     * &lt;code&gt;CC 0000&lt;/code&gt;, &lt;code&gt;CC 0020&lt;/code&gt;, &lt;code&gt;JCL ERROR&lt;/code&gt;, &lt;code&gt;ABEND S0C4&lt;/code&gt; etc.&lt;br&gt;

```

Comment:
Nope. `<code>123</code>` renders the numbers 123 in the monospace font.
You are rendering them in the normal font with `<` and `>` characters.
Needs reverting I think.

Reply:
or use quotes.